tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc

```diff
@@ -777,7 +783,8 @@ Status ConvertAfterShapes(const ConversionParams& params) {
     engine_segments.push_back(std::move(curr_engine));
     converted_segments.push_back(std::move(curr_segment));
 
-    if (VLOG_IS_ON(8)) {
+    if (VLOG_IS_ON(8) && 
+        curr_engine.engine_type == EngineInfo::EngineType::TRTStatic) {
```

Comment:
Why only static?

Reply:
I thought this was because we only serialize in static mode, but I now realize this is a logging functionality.